DemoApp/StreamChat/Components/CustomAttachments/DemoQuotedChatMessageView.swift

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
```

Comment:
Make sure to update the new files with the new year before merging.

Reply:
Done 👍 I've also updated the Xcode template for upcoming files